Project/Assets/Prairie/Framework/Script/Annotation/ParseAnnotation.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+public class ParseAnnotation {
+
+    private static readonly string IMAGE_TEXT_FULL = "<\\s*img\\s*=\\s*\"?.*?\"?\\s*>";
+    private static readonly string IMAGE_TEXT_SPLIT = "(<\\s*img\\s*=\\s*\"?)|(\"?\\s*>)";
+
+    /// <summary>
+    /// Takes a string and parses text so that textOut gets populated by strings, separated by image paths that are stored in imgOut
+    /// </summary>
+    /// <param name="text"></param>
+    /// <param name="textOut"></param>
+    /// <param name="imgOut"></param>
+    public static void ParseAnnotationText(string text, List<string> textOut, List<string> imgOut)
```

Comment:
Hmm. Possible. Let me think about it when I can look at the code again (I'm on my phone at work right now)/ maybe we can discuss it tonight at the meeting (not sure if I'll have time to look before then)